lib/transport_web/templates/layout/_navigation.html.eex

```diff
@@ -1,5 +1,9 @@
 <nav id="navigation" role="navigation">
   <div class="navigation-wrapper">
     <%= img_tag("images/logo.svg", alt: "transport.beta.gouv.fr") |> link(to: "/", class: "logo") %>
+
+    <a style="float:right; color: white; margin: 1em;" href="<%= session_path @conn, :new %>">
```

Comment:
Just an example, not to merge like that ! Also, we need to add a previous step where we explain they'll be redirected.